buf.gen.yaml

```diff
@@ -0,0 +1,9 @@
+version: v1
+plugins:
+  - plugin: go
+    out: .
+    opt: paths=source_relative
+  - name: go-grpc
+    out: .
+    opt:
+      - paths=source_relative
```

Comment:
Since this is all OSS we could use the remote plugins eventually, but it doesn't need to be part of this PR.